src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  margin: 0 50px;
+  justify-content: space-between;
```

group your properties

```suggestion
  display: flex;
  justify-content: space-between;
  margin: 0 50px;
```